utils/pretty_printers.cc

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include "pretty_printers.hh"
+
+namespace utils {
+
+std::ostream& operator<<(std::ostream& os, pretty_printed_data_size data) {
+    static constexpr const char *suffixes[] = {" bytes", "kB", "MB", "GB", "TB", "PB"};
```

Comment:
```suggestion
    static constexpr const char *suffixes[] = {" bytes", "KB", "MB", "GB", "TB", "PB"};
```

Reply:
Q: why do we prefer those over :

KiB - Kibibyte (1024 bytes) |   | MiB - Mebibyte (1024 KiB) |   | GiB - Gibibyte (1024 MiB) |   | TiB - Tebibyte (1024 GiB)

(which then allow division by 1024 and using shift for this) ?

Reply:
Regardless, such change should be done in a separate PR (and probably won't get backported).

Reply:
Agreed.

Reply:
kB is more correct than KB.